inc/querysubquery.class.php

```diff
@@ -74,4 +74,8 @@ public function getQuery() {
       }
       return $sql;
    }
+
+   public function __toString() {
```

Comment:
I guess it should be implemented in:
- AbstractQuery
- QueryParam
- QueryExpression